.github/workflows/lint.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     - name: Run make indent
       run: >
         if [ -z "${{github.base_ref}}" ]; then
+          git fetch --deepen=1 &&
```

Comment:
Perhaps we can use
```yaml
- uses: actions/checkout@v3
    with:
        fetch-depth: 0  # all history for all branches and tags

Reply:
1) I believe it would be heavier load on ci infra to get full history instead of just two last commits.
2) I intentionally didn't use `fetch-depth` as in case github.base_ref is set e.g. for `pull_request` action, github already fetches all commits of the pull request unlike `push` action, so with simple git-fetch-deepen I can only fetch additional commits when needed.